src/pages/Home/home.m.scss

```diff
@@ -0,0 +1,106 @@
+@import 'styles/mixins';
+@import 'styles/colors';
+
+.home {
+  @include box;
+
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.title {
+  padding: 54px 0 24px;
+  font-size: 24px;
+  line-height: 34px;
+  color: $main-black;
+  font-weight: normal;
+}
+
+.transfer-buttons {
```

Comment:
why not just `buttons` or `list` or `actions`, whatever in one word?

Reply:
Because there are multiple attention locuses on the page, and I need to specify which area is it about. I don't know now whether we would have other buttons on page encapsulated into separate components or not (it's depends of specific implementation a lot), so I made transfer buttons separated by default.

I will rename them, but I think it's ok to have namespaced elements if they are in common context.

Btw, WebStorm now supports converting `kebab-case-class-names` to `camelCaseClassNames` for CSS modules in suggestions with zero configuration. Try it!))